base/logging_test.go

```diff
@@ -55,6 +55,8 @@ func TestRedactedLogFuncs(t *testing.T) {
 
 func Benchmark_LoggingPerformance(b *testing.B) {
 
+	defer SetUpBenchmarkLogging(LevelInfo, KeyAll)()
```

Comment:
Set default+CRUD log key at the start

```suggestion
	defer SetUpBenchmarkLogging(LevelInfo, KeyHTTP|KeyCRUD)()
```

and remove `consoleLogger.LogKey.Enable` line below